### DISCUSSION TITLE
Player Indicator Highlight Regardless of Online Status

### DISCUSSION THREAD

**Discussion by CleetZ** [runelite/runelite#16725] [Ideas]
Hello. Could You guys please consider making it so players get highlighted regardless if they are online or not, if you have them added to friend list? I want to use player indicator to highlight bots so they are easier to kill. Thanks for Your time and consideration!

**Comment by Felanbird** [comment]
Discussed all the way back in 2018 and the consensus was that the game setting should be respected.
https://github.com/runelite/runelite/pull/951#issuecomment-379238853
